includes/footer.php

```diff
@@ -1,3 +1,32 @@
 <!-- START FOOTER -->
+    <footer>
+        <div class="row">
+            <a href="/" class="one column"><i class="far fa-envelope"></i></a>
+            <a href="/" class="one column"><i class="fas fa-mobile-alt"></i></a>
+            <a href="/" class="one column"><i class="fas fa-question-circle"></i></a>
+            <a href="/" class="one column"><i class="fas fa-user-circle"></i></a>
+
+            <p style="background-color: #2E9E47" class="four columns">Treat Street</p><!--replace with image-->
```

Comment:
For now the inline-css is ok but we'll need to update this later on

Reply:
Okay. Got it!